docs/swupd.1.rst

```diff
@@ -303,6 +303,17 @@ SUBCOMMANDS
         Like --picky, but it only looks for extra files. It omits checking
         hash values, and for missing files, directories and/or symlinks.
 
+    - `--bundles`
+
+      Forces swupd to only diagnose the (comma separated) list of bundles
+      provided.
+
+      Examples:
+
+        - ``--bundles xterm,vi``
+
+            Diagnoses only bundles os-core and vi.
```

Comment:
vi and xterm?

Reply:
🤣 good catch @karthikprabhuvinod , it should be xterm and vi